ui/src/components/Atoms/User/User.tsx

```diff
@@ -92,19 +92,25 @@ const Customer: React.FC<UserProps>= (props) => {
         handleChangeShown()
         handleCleanState(true)
       }}
-      buttonStyle={styles.dropDownBtn}/>
+      buttonStyle={`${styles.dropDownBtn}`}/>
     ]
 
     const submitButton = isLoading
     ? <Progress key="key" containerStyles={styles.dropdownSpinner} />
-    : <Button key="button-Recipient" label={"Submit"} onClick={()=> {
+    : <Button key="button-Recipient" label={"Submit"} id='test-invoice-submit' onClick={()=> {
```

Comment:
Minor: don't feel the need to add the prefix. This applies to other places as well.
```suggestion
    : <Button key="button-Recipient" label={"Submit"} id='invoice-submit' onClick={()=> {
```

Reply:
yeah, made more sense with the classes.
keeping them kind of clarifies the reason (which is not enforced though, so no guarantee here)
so if you don't mind, let's keep the prefix for now